http/src/request/guild/member/update_guild_member.rs

```diff
@@ -127,6 +131,14 @@ impl<'a> UpdateGuildMember<'a> {
         self
     }
 
+    /// Sets when the user's time out will expire and be capable of communicating
+    /// in the guild again. Set to `None` to remove the time out completely.
```

Comment:
```suggestion
    /// Set the member's [Guild Timeout].
    ///
    /// The timestmap indicates when the user will be able to communicate again.
    /// It can be up to 28 days in the future. Set to [`None`] to remove the
    /// timeout. Requires the [`MODERATE_MEMBERS`] permission.
    ///
    /// [Guild Timeout]: https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ
    /// [`MODERATE_MEMBERS`]: twilight_model::guild::Permissions::MODERATE_MEMBERS

```